DI-435/find_vcfs_to_merge.py

```
@@ -409,84 +473,136 @@ def main():
     projects_to_print = '\n\t'.join([
         f"{x['describe']['name']} - {x['id']}" for x in b38_projects
     ])
-    print(f"\n{len(b38_projects)} projects found:\n\t{projects_to_print}")
-
-    # Get QC status files from b37 projects and read them in
-    all_qc_files, missing_projects, b38_project_subset = get_qc_files(
-        b38_projects, args.start, args.end
+    print(
+        f"\n{len(b38_projects)} GRCh38 projects found:\n\t{projects_to_print}"
```

Ruff (0.8.0)

Trailing comma missing

Add trailing comma

(COM812)